geometry_msgs/Pose pose
float64 dt
---
bool result
